en_US/changes/changes-ee-4.4.md

```diff
@@ -1,5 +1,27 @@
 # Releases
 
+## e4.4.21
+
+*Release Date: 2023-10-13*
+
+### Enhancements
+
+- Added support for Confluent data bridge.
+
+- Now the MQTT topic field in the Kafka consumer group supports templates with placeholders.
+
+  For example, if the key of the message consumed by Kafka is "a", and the configured MQTT topic is "topic/${key}", then when the message is forwarded, the MQTT topic will be replaced with "topic/a".
+
+- Now the "Message Republish" action supports adding MQTT properties and user properties.
+
+  The "Message Republish" action supports two new fields: "MQTT Properties" and "User Property Value", both of which are in the format of key-value pairs, and both the key and value support placeholders.
```

Comment:
<img width="1031" alt="image" src="https://github.com/emqx/emqx-docs/assets/67890146/b0ba689c-66b0-42aa-b49b-351b15b033f9">
MQTT Properties -> mqtt_properties_template
User Property Value -> user_properties_template

Reply:
Thanks! Now you are not just a technical writer but are an emqx expert!